TACC/Frontera-Portal
# bug/FP-860: Get job output location from notification info

## Overview: ##

Seems the _links.archiveData.href no longer contains the info we need to show a job's archive path.
This gets the job archive path from notifications, and sticks it into the job listing state.

## Related Jira tickets: ##

* [FP-860](https://jira.tacc.utexas.edu/browse/FP-860)

## Summary of Changes: ##
- get job output location from notification info
- remove right-to-left string alignment for job path

## Testing Steps: ##
1. Go to the dashboard or history views
2. Ensure you have job history, or run some jobs. You'll need to setup ngrok:
run `ngrok http 443` and set that value as `_WH_BASE_URL`
3. Confirm that, even on page refresh, the job output path field is populated

## UI Photos:
<img width="871" alt="Screen Shot 2021-01-20 at 4 34 39 PM" src="https://user-images.githubusercontent.com/20326896/105249241-69238400-5b3d-11eb-8b32-e065d898a243.png">
